scripts/devoverride.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Suggest to use env to find `bash` because it's not always in `/bin` (some systems have it in `/usr/local/bin` for example).

```suggestion
#!/usr/bin/env bash
```